webapp/src/shared/components/templates/AvatarPageTemplate/AvatarPageTemplate.tsx

```diff
@@ -23,6 +28,20 @@ type AvatarPageTemplateProps = {
   secondaryButton?: ButtonProps;
 };
 
+type AvatarPageRegularTemplateProps = AvatarPageTemplateCommonProps & {
+  variant?: AvatarPageVariants;
+  captionLikeElement?: never;
+};
+
+type AvatarPageCaptionLikeTemplateProps = AvatarPageTemplateCommonProps & {
+  variant: AvatarPageVariants;
+  captionLikeElement: JSX.Element;
+};
+
+type AvatarPageTemplateProps =
+  | AvatarPageRegularTemplateProps
+  | AvatarPageCaptionLikeTemplateProps;
+
```

Comment:
suggestion: What do you think about adding the prop `captionLikeElement?: JSX.Element;` to the `AvatarPageTemplateProps`. This way, we could remove the variant and do something like this:

```tsx
          <div className="avatar-page-caption-like-wrapper">
            {captionLikeElement && captionLikeElement}
          </div>
```

Reply:
done. Thx!